WeakAuras/RegionTypes/Texture.lua

```diff
@@ -21,6 +21,9 @@ local default = {
   yOffset = 0,
   frameStrata = 1
 };
+if WeakAuras.IsClassic then
```

Comment:
So, in the case of this kind of thing, I think i would actually prefer if the look and feel of newly minted auras didn't change between classic and retail. So, I'd prefer that we just use a texture available in both clients (or one of our own) as the region default.